lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -71,7 +82,9 @@ defmodule Trento.Clusters do
           :ok
 
         {:error, reason} ->
-          Logger.error("Failed to request checks execution for cluster #{cluster_id}: #{reason}")
+          Logger.error(
```

Comment:
Not something you should do in this PR, but we should reduce the errors in this `each` now and return error if any of them fails.
Or this return the error if fails?

Reply:
this is used by the job scheduler in a best-effort manner, this is why we just use the each loop and we return nothing.
If something went wrong while scheduling the execution we log it